### DISCUSSION TITLE
ModuleNotFoundError: No module named 'pytorch_lightning.utilities.debugging'

### DISCUSSION THREAD

**Discussion by lynnhio** [Lightning-AI/pytorch-lightning#13063] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Recently, I'm changing my envir from 1 GPU to multi-GPU environment.

When I rerun my code in the new envir, it shows **ModuleNotFoundError: No module named 'pytorch_lightning.utilities.debugging'.** from codeline: **model = torch.load(self.model_path)**

I saved model using torch.save("bestmodel.pkl") and used torch.load() as usual.
Can't find this pytorch_lightning.utilities.debugging. did anybody bump into the same one?

version info:
- torch version 1.11.0
- pytorch_lightning version 1.6.3

model info:
- BERT NLP classification model

Can anyone offer some advices?

thanks.

**Comment by rohitgr7** [comment]
can you share the complete stacktrace?
